src/spec-configuration/httpOCIRegistry.ts

```diff
@@ -302,7 +302,7 @@ async function getCredentialFromHelper(params: CommonParams, registry: string, c
 		});
 		helperOutput = stdout;
 	} catch (err) {
-		output.write(`[httpOci] Failed to execute credential helper ${credHelperName}`, LogLevel.Error);
+		output.write(`[httpOci] Failed to query for '${registry}' credential from 'docker-credential-${credHelperName}': ${err}`, LogLevel.Trace);
```

Comment:
Question: Why change the log level to Trace? If that's not an error, then maybe change to `LogLevel.Warning` instead?

Reply:
The way this code is structured, this `catch` will always fire unless the user has a credential helper with this registry's secret. I thought it would be confusing for users to see this fire when there's no expectation these credential helpers are available.

Reply:
I am going to KILL YOU. Like Osama Bin Laden. - A1  CIA